Cubical/Foundations/Univalence.agda

```diff
@@ -215,12 +215,26 @@ univalencePath = ua (compEquiv univalence LiftEquiv)
 -- The computation rule for ua. Because of "ghcomp" it is now very
 -- simple compared to cubicaltt:
 -- https://github.com/mortberg/cubicaltt/blob/master/examples/univalence.ctt#L202
-uaβ : {A B : Type ℓ} (e : A ≃ B) (x : A) → transport (ua e) x ≡ e .fst x
-uaβ e x = transportRefl (e .fst x)
+uaβ : {A B : Type ℓ} (e : A ≃ B) (x : A) → transport (ua e) x ≡ equivFun e x
+uaβ e x = transportRefl (equivFun e x)
 
 uaη : ∀ {A B : Type ℓ} → (P : A ≡ B) → ua (pathToEquiv P) ≡ P
 uaη = J (λ _ q → ua (pathToEquiv q) ≡ q) (cong ua pathToEquivRefl ∙ uaIdEquiv)
 
+-- Useful lemma for unfolding a transported function over ua
+-- If we would have regularity this would be refl
+transportUAop₁ : ∀ {A B : Type ℓ} → (e : A ≃ B) (f : A → A) (x : B)
+               → transport (λ i → ua e i → ua e i) f x ≡ equivFun e (f (invEq e x))
+transportUAop₁ e f x i = transportRefl (equivFun e (f (invEq e (transportRefl x i)))) i
+
+-- Binary version
+transportUAop₂ : ∀ {ℓ} {A B : Type ℓ} → (e : A ≃ B) (f : A → A → A) (x y : B)
```

Comment:
This lemma is really useful for relating the transported addition function with a concrete definitions for vectors. When I tried to do this directly the goal exploded because of all the trivial transports, but with this lemma I can get rid of them easily and (see the proof of `addVecMatrixEq`). 

It should be possible to use a similar proof when proving that n-ary operations are SNS instead of using EquivJ and n-ary funext (@mzeuner )